ports/stm32/Makefile

```diff
@@ -139,7 +140,7 @@ LDFLAGS += --gc-sections
 # Debugging/Optimization
 ifeq ($(DEBUG), 1)
 CFLAGS += -g -DPENDSV_DEBUG
-COPT = -Og
+COPT = -Os
```

Comment:
This shouldn't be changed.

Reply:
Ah, during debugging I changed it here... Changed it. Should I somehow revert to the base, or is a commit with the correct value OK?